screens/Registration/AuthenticationScreen/AuthenticationScreen.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { View, Text, Button, StyleSheet } from 'react-native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import { peach, white } from '../../../assets/colors/colors';
+
+// Define your navigation stack parameter list
+type AuthStackParamList = {
+  Authentication: undefined;
+  Information: undefined;
+  // Add other routes and parameters here as needed
+};
+
+// Define the navigation prop type based on the stack parameter list
+type AuthenticationScreenNavigationProp = StackNavigationProp<AuthStackParamList, 'Authentication'>;
+
+// Define the component prop types
+type AuthenticationScreenProps = {
+  navigation: AuthenticationScreenNavigationProp;
+};
```

Comment:
Since the type definition is not re-used anywhere, you can simplify the code a bit.

```suggestion
// Define the component prop types
type AuthenticationScreenProps = {
  navigation: StackNavigationProp<AuthStackParamList, 'Authentication'>;
};
```